x-pack/plugins/security_solution/cypress/integration/alerts_detection_rules.spec.ts

```diff
@@ -28,43 +27,45 @@ import {
   waitForLoadElasticPrebuiltDetectionRulesTableToBeLoaded,
   waitForRuleToBeActivated,
 } from '../tasks/alerts_detection_rules';
-import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
 import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
 import { DEFAULT_RULE_REFRESH_INTERVAL_VALUE } from '../../common/constants';
 
 import { DETECTIONS_URL } from '../urls/navigation';
-import { removeSignalsIndex } from '../tasks/api_calls/rules';
+import { createCustomRule, removeSignalsIndex } from '../tasks/api_calls/rules';
 import { cleanKibana } from '../tasks/common';
+import { existingRule, newOverrideRule, newRule, newThresholdRule } from '../objects/rule';
 
 describe('Alerts detection rules', () => {
-  before(() => {
+  beforeEach(() => {
     cleanKibana();
     removeSignalsIndex();
-    esArchiverLoad('prebuilt_rules_loaded');
+    loginAndWaitForPageWithoutDateRange(DETECTIONS_URL);
+    waitForAlertsPanelToBeLoaded();
+    waitForAlertsIndexToBeCreated();
+    createCustomRule(newRule, 'rule1');
+    createCustomRule(existingRule, 'rule2');
+    createCustomRule(newOverrideRule, 'rule3');
+    createCustomRule(newThresholdRule, 'rule4');
   });
 
   after(() => {
-    esArchiverUnload('prebuilt_rules_loaded');
+    cy.clock().invoke('restore');
```

Comment:
Nit: this teardown is specific to a single time-traveling test. It would be nice if such tests were isolated in their own context with clock-specific setup/teardown hooks.